asset/css/markbind.css

```diff
@@ -224,7 +224,7 @@ footer {
 
 /* Temporary Anti-FOUC Styles */
 
-/* By default, some VueStrap components break the layout of the page before Vue is mounted,
+/* By default, some vue components break the layout of the page before Vue is mounted,
```

Comment:
```suggestion
/* By default, some MarkBind Vue components break the layout of the page before Vue is mounted,
```

Reply:
pushed 2 more fixups (one for the test files commit) to update this to `some of MarkBind's Vue`